src/FritzBox/Converter.php

```diff
@@ -145,53 +145,52 @@ private function getPhoneNumbers(stdClass $card): array
             return [];
         }
 
-        $phoneNumbers = [];
+        $addNumber = [];
+        $idnum = -1;
```

Comment:
Ich möchte die Variable gerne loswerden. Lässt die sich wirklich nicht woanders ableiten?

Reply:
Ich nehme an du meinst $idnum? Ich habe keinen anderen Weg gefunden durchgängig in der Funktion das (aktuell richtige) Element im Array korrekt zu adressieren